website/static/js/fangorn.js

```diff
@@ -509,12 +532,7 @@ function doItemOp(operation, to, from, rename, conflict) {
 
         Raven.captureMessage('Failed to move or copy file', {
             xhr: xhr,
-            requestData: {
-                rename: rename,
-                conflict: conflict,
-                source: waterbutler.toJsonBlob(from),
-                destination: waterbutler.toJsonBlob(to),
-            }
+            requestData: jsonData
```

Comment:
Oh, I see what you mean when you were asking that question about raven now.  `requestData` should be an object, not a serialized string.  Consider leaving `jsonData` as an object and only serializing it when setting the `POST` body.